components/pedal-details-styles.styled.js

```diff
@@ -0,0 +1,31 @@
+import styled from "styled-components";
+
+export const StyledDetailMain = styled.main`
+  background-color: antiquewhite;
+  color: black;
+`;
+
+export const StyledDetailHead = styled.header`
+  background-color: darkgray;
+  width: 100%;
+  margin: 0;
+  padding: 10px, 10px, 10px, 20px;
+  display: flex;
+  justify-content: space-between;
+  //box-shadow: 0px 10px 6px 0px #2596be;
```

Comment:
If you don't need this, highly recommend removing it. No need to have comment out code in produciton.